sphinxcontrib/chapeldomain/__init__.py

```diff
@@ -617,7 +620,7 @@ def get_index_text(self, modname, name_cls):
                 return _('%s() (built-in %s)') % \
                     (name_cls[0], self.chpl_type_name)
             return _('%s() (in module %s)') % (name_cls[0], modname)
-        elif self.objtype in ('data', 'type', 'enum'):
+        elif self.objtype in ('data', 'type'):
```

Comment:
I think this is here because you can declare enums inside a class as well as outside (see `test/classes/bradc/enumInClass.chpl`, which appears to work today).  So I think it would be good to keep this, or at least write a test case that does so and ensure it works the same before and after your change (and maybe ensure you can still follow links to it).

Reply:
Well that definitely seems like an improvement